app/src/routes/accept-invite.vue

```diff
@@ -47,6 +48,10 @@ async function onSubmit() {
 		creating.value = false;
 	}
 }
+
+useHead({
+	title: t('invite'),
```

Comment:
Opted for "Create Account" since that is also how the form is presented to the user:
<img width="300" src="https://github.com/directus/directus/assets/5363448/5195f077-4d52-461f-a343-de166549cc36">


```suggestion
	title: t('create_account'),
```